SignalMessaging/categories/Theme.m

```diff
@@ -110,7 +110,12 @@ + (UIColor *)toolbarBackgroundColor
 
 + (UIColor *)cellSelectedColor
 {
-    return (Theme.isDarkThemeEnabled ? UIColor.ows_whiteColor : UIColor.ows_blackColor);
+    return (Theme.isDarkThemeEnabled ? [UIColor colorWithWhite:0.2 alpha:1] : [UIColor colorWithWhite:0.92 alpha:1]);
+}
+
++ (UIColor *)cellSeparatorColor
```

Comment:
Why is this in the Theme if it doesn't depend on the theme state?  I suggest `UIColor+OWS`.